LibsAndSamples.sln

```diff
@@ -1502,6 +1506,34 @@ Global
 		{DD23D8FF-86BA-4E9F-8AF1-0EBE0D86986A}.Release|x64.Build.0 = Release|Any CPU
 		{DD23D8FF-86BA-4E9F-8AF1-0EBE0D86986A}.Release|x86.ActiveCfg = Release|Any CPU
 		{DD23D8FF-86BA-4E9F-8AF1-0EBE0D86986A}.Release|x86.Build.0 = Release|Any CPU
+		{55854CE5-923F-47E4-8904-9F198C8E2A7F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
Please read below. Let's not add a new project for this. We should also not add all these platforms to the SLN.

Reply:
Done. Removed